### ISSUE TITLE
non-JVM implementation

### ISSUE THREAD

**Issue by korya** [awslabs/amazon-kinesis-client#45]
Is there a non-JVM alternative to the project? An official one? or a community project?

**Comment by pfifer** [comment]
At this time there isn't a non-JVM alternative.  We are always investigating other possibilities to add these features.

Please reopen if you have any additional questions.

**Comment by jqr** [comment]
@korya I've considered writing one in Ruby to easily side-stop multi-process spawning of the MultiLangDaemon, and because the pattern isn't *that* complicated. Can you explain why you're interested in a non-JVM version?

**Comment by korya** [comment]
The main reason to avoid using JVM because it is a huge pain to maintain it. Of course if your entire backend stack is based on JVM. In our case, we try to use more "lightweight" technologies: go, nodejs, php. This makes very easy to develop, build, deploy and replace our backend code. Anyway for now we have decided not to use kinesis. We considered it as an option to quickly start using event streams, but found it too complex for our simple needs.

I agree that it should not be too hard to write a custom client. But we found that it is not worth the effort.
